### PR TITLE
Gov store olp 1042

### PR DESCRIPTION
**Pull request by kingpinXD**

### CODE REVIEW

File: app/application.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func (app *App) setupState(stateBytes []byte) error {
 	if err != nil {
 		return errors.Wrap(err, "Error in setting up ONS options")
 	}
-	app.Context.domains.SetOptions(&initial.Governance.ONSOptions)
+	//app.Context.domains.SetOptions(&initial.Governance.ONSOptions)
```
Comment (ws4charlie) [inline app/application.go:147]: You can remove it if it's redundant.

File: app/application.go [hunk 2]
```diff
@@ -357,11 +357,11 @@ func (app *App) Prepare() error {
 
 		app.Context.feePool.SetupOpt(feeOpt)
 
-		onsOpt, err := app.Context.govern.WithHeight(app.header.Height).GetONSOptions()
-		if err != nil {
-			return err
-		}
-		app.Context.domains.SetOptions(onsOpt)
+		//onsOpt, err := app.Context.govern.WithHeight(app.header.Height).GetONSOptions()
```
Comment (ws4charlie) [inline app/application.go:360]: You can remove it if it's redundant.

File: app/context.go [hunk 3]
```diff
@@ -247,7 +247,7 @@ func (ctx *context) Services() (service.Map, error) {
 	ethTracker.SetupOption(ctx.ethTrackers.GetOption())
 
 	onsStore := ons.NewDomainStore("d", storage.NewState(ctx.chainstate))
-	onsStore.SetOptions(ctx.domains.GetOptions())
+	//onsStore.SetOptions(ctx.domains.GetOptions())
```
Comment (ws4charlie) [inline app/context.go:250]: Same.

File: app/application.go [hunk 4]
```diff
@@ -203,10 +202,10 @@ func (app *App) setupState(stateBytes []byte) error {
 	for _, domain := range initial.Domains {
 		if ons.GetNameFromString(domain.Name).IsValid() && app.Context.domains.GetOptions().IsNameAllowed(ons.Name(domain.Name)) {
 			d, err := ons.NewDomain(domain.Owner, domain.Beneficiary, domain.Name, 0, domain.URI, domain.ExpireHeight)
-			d.ActiveFlag = false
 			if err != nil {
 				return errors.Wrap(err, "failed to create initial domain")
 			}
+			d.ActiveFlag = false
```
Comment (kingpinXD) [inline app/application.go:208]: Moved the set flag after error check

**Pull request merged**